defaults/main.yml

```diff
@@ -1,3 +1,10 @@
 ---
 scylladb_repo_url: ""
 scylladb_version: ""
+scylladb_cluster_name: ""
+scylladb_seed_servers: ""
+scylladb_developer_mode: "true" # This should be set to false for production
+
+scylladb_jmx:
```

Comment:
Do we have to touch this file / config?  These seem like defaults to me unless I'm incorrect.

Reply:
Removed